com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/PathTracing/Shaders/PathTracingBSDF.hlsl

```diff
@@ -387,6 +461,25 @@ bool SampleDelta(MaterialData mtlData,
     return any(outgoingDir);
 }
 
+bool SampleLambert(MaterialData mtlData,
+                   float3 inputSample,
+               out float3 outgoingDir,
+               out float3 value,
+               out float pdf)
+{
+    bool retVal = BRDF::SampleLambert(mtlData, inputSample, outgoingDir, value, pdf);
+    outgoingDir *= -1.0;
```

Comment:
why is there a -1.0? can you comment?

Reply:
Well it's just a flip, I'll write it as x = -x;